components/network/proftpd/Makefile

```diff
@@ -20,45 +20,47 @@
 #
 
 #
-# Copyright (c) 2011, 2014, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2011, 2019, Oracle and/or its affiliates. All rights reserved.
 #
 
+BUILD_BITS=64
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		proftpd
-COMPONENT_VERSION=	1.3.5
-COMPONENT_REVISION=	5
-COMPONENT_SUBVERSION=	d
+COMPONENT_VERSION=	1.3.6
+COMPONENT_REVISION=	6
```

Comment:
You can remove it as COMPONENT_VERSION increased.

Reply:
Fixed.